packages/core/src/card/card.stories.ts

```diff
@@ -0,0 +1,153 @@
+
+const placeholder = 'Placeholder';
+const basicContent = `
+    Message: Can you imagine what we will be downloading in another twenty years?
+    Who would have ever thought that you could record sound with digital quality fifty years ago?
+    Now we routinely download whole albums worth of music in a couple of minutes.
+  `;
+const longerContent = `
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    Praesent volutpat tortor eget quam auctor, quis sagittis libero auctor.
+    Nulla augue ante, tincidunt sit amet semper vitae, tempus at ipsum.
+    Vestibulum elementum, turpis quis ullamcorper fermentum, elit turpis placerat ipsum, quis convallis ex nisi sit amet lacus.
+    Ut enim ipsum, tincidunt nec luctus id, pharetra id velit. Aliquam nec elit ut neque lacinia mattis id ac lorem.
+    Vivamus egestas massa nulla, ac elementum purus pretium eu. Duis ultrices nec tortor varius consectetur.
+    Maecenas convallis bibendum urna, et aliquam quam tincidunt in.
+    Nam viverra, leo vitae imperdiet ultrices, ante mi facilisis mi, ac vehicula erat nulla sed nibh.
+    Nam euismod gravida est, convallis placerat nunc pharetra sed. Nullam scelerisque dui augue, eu porta erat tempus in.
+
+    Integer egestas massa orci, id scelerisque libero lobortis vitae.
+    Vivamus aliquet sem massa, eget sodales sapien dictum eu.
+    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Suspendisse non interdum sem, nec egestas nulla.
+    Duis vitae leo justo. Fusce ante enim, tincidunt nec faucibus sed, fringilla et ipsum.
+    Proin quis diam elit. Cras nec finibus libero. Sed bibendum lectus eget magna ultricies eleifend.
+    Duis ut molestie urna. Proin cursus dolor ac ex rutrum, eu condimentum elit tincidunt.
+    In a dolor faucibus, vehicula ipsum eu, viverra lorem. Aenean congue fermentum commodo.
+    Sed ex purus, consectetur mollis urna eget, vestibulum mollis dolor.
+    Donec sodales, felis sit amet gravida ornare, magna dolor ullamcorper turpis, vel facilisis odio ipsum semper sem.
+  `;
+
+export default {
+  title: 'Stories/Card',
+  component: 'cds-card',
+  argTypes: getElementStorybookArgTypes('cds-card', customElements),
+  parameters: {
+    options: { showPanel: true },
+  },
+};
+
+export function Basic(args: { [key: string]: unknown }) {
+  return html`<cds-card ...="${spreadProps(getElementStorybookArgs(args))}">
+    <p cds-text="p1">${placeholder}</p>
+  </cds-card>`;
+}
+
+export function WithLayout(args: { [key: string]: unknown }) {
+  return html` <cds-card ...="${spreadProps(getElementStorybookArgs(args))}">
+    <div cds-layout="vertical">
+      <div cds-layout="p-t:xmd p-l:xmd">
+        <p cds-text="section">Card Title</p>
+      </div>
+
+      <div cds-layout="p:xmd">
+        <p cds-text="p1">${basicContent}</p>
```

Comment:
We should update this to use `body` rather than `p1`. The p and h utils are temporary to help migration and will eventually go away. We want to encourage teams to use the semantic utilities when possible.